lms/djangoapps/teams/static/teams/js/views/search_field.js

```diff
@@ -0,0 +1,58 @@
+;(function (define) {
+    'use strict';
+
+    define(['backbone', 'underscore', 'text!teams/templates/search-field.underscore'],
+        function (Backbone, _, searchFieldTemplate) {
+            return Backbone.View.extend({
+
+                events: {
+                    'submit .teams-search-form': 'performSearch',
+                    'keyup .search-field': 'refreshState',
+                    'click .cancel-button': 'clearSearch'
+                },
+
+                initialize: function(options) {
+                    this.collection = options.collection;
```

Comment:
This line isn't really necessary -- Backbone sets `['model', 'collection', 'el', 'id', 'attributes', 'className', 'tagName', 'events']` automatically on views when they're passed in as options.